### ISSUE TITLE
compare not working in version 0.3.8

### ISSUE THREAD

**Issue by VRTOlli** [tacoss/testcafe-blink-diff#6]
Hi, 

I tried to compare the base and actual images from my tests, but I kept getting an error that 'reporter.iife.min.js' was missing.
[ERR] ENOENT: no such file or directory, open '/home/olvahakk/projects/VRTfront/node_modules/testcafe-blink-diff/dist/reporter.iife.min.js'

It seems that the whole dist folder was missing. I switched to 0.3.7 and it worked just fine.

**Comment by pateketrueke** [comment]
You're right, I'm gonna fix this asap, thank you!

**Comment by gazab** [comment]
Why closed? This is still an issue.

**Comment by pateketrueke** [comment]
Sorry for that, I release 0.3.9 can you confirm is still an issue?

**Comment by gazab** [comment]
Yeah, the reporter is still missing unfortunately.

Version 0.3.6:
`$ ls node_modules/testcafe-blink-diff/dist/`
`reporter.iife.min.js`

Version 0.3.7:
`$ ls node_modules/testcafe-blink-diff/dist/`
`reporter.iife.min.js`

Version 0.3.8:
`$ ls node_modules/testcafe-blink-diff/dist/`
`ls: cannot access 'node_modules/testcafe-blink-diff/dist/': No such file or directory`

Version 0.3.9:
`$ ls node_modules/testcafe-blink-diff/dist/`
`ls: cannot access 'node_modules/testcafe-blink-diff/dist/': No such file or directory`

**Comment by gazab** [comment]
So you get this error when you run it:
```
> testcafe-blink-diff snapshots --compare base:actual --threshold 0.03

Collecting screenshots...
Processing 1 file...
  button DONE
[ERR] ENOENT: no such file or directory, open '<PATH_TO_APP>\node_modules\testcafe-blink-diff\dist\reporter.umd.js'
```

**Comment by pateketrueke** [comment]
I found the reason and comitted the same mistake twice, sorry, I just released `0.3.11` fixing this issue now. 😀

**Comment by gazab** [comment]
Works like a charm! Thanks!
